WordPress/src/main/java/org/wordpress/android/ui/history/HistoryDetailsContainerFragment.java

```diff
@@ -71,9 +75,25 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         return rootView;
     }
 
-    @Override
-    public void onViewCreated(final FullScreenDialogController controller) {
-        mDialogController = controller;
+    @Override public void onCreate(@Nullable Bundle savedInstanceState) {
```

Comment:
```suggestion
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
```

Can we move the `@Override` annotation to a separate line for code style consistency?